### DISCUSSION TITLE
Dynamic height using drag'n drop and subflows

### DISCUSSION THREAD

**Discussion by brunomsa** [xyflow/xyflow#2490] [Q&A]
Hello, I'm trying to use react-flow to build flows that the user can drag and drop nodes to subflows. I've looked, but I haven't found anything related to dragging to a subflow and adjusting the height of that subflow automatically. And as it is a vertical flow, it is also necessary to adjust the position (x,y) of the nodes that are outside the subflow. Is there already something related that I can use?

**Comment by antipasvasil** [comment]
Hi @brunomsa maybe this here https://reactflow.dev/docs/examples/interaction/drag-and-drop/ can help somehow

**Reply by brunomsa** [reply]
It helped me a bit, but it's still not what I need. For my case, I was needing something more similar to the behavior of this lib. You can see that when you drag a node into the subflow, the subflow adjusts the height dynamically.
[https://reaflow.dev/?path=/story/demos-drag--nested-node-rearranging](url)
